app/models/vacation.rb

```diff
@@ -0,0 +1,5 @@
+class Vacation < ApplicationRecord
+  validates :started_on, presence: true, date: true
```

Comment:
remove the possibility of creating a vacation with a `started_on` after `finished_on`